src/rev_ai/apiclient.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""Speech recognition tools for using rev.ai
+"""
```

Comment:
minor -  strange line break please remove

Reply:
this can be normal for python, I'm a promoter of this style but I don't enforce it on anyone